Grail Diary/Grail Diary/POIDetailViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  POIDetailViewController.swift
+//  Grail Diary
+//
+//  Created by macbook on 9/12/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class POIDetailViewController: UIViewController {
+    
+    @IBOutlet weak var locationLabel: UILabel!
+    @IBOutlet weak var countryLabel: UILabel!
+    @IBOutlet weak var cluesTextView: UITextView!
+    
+    var poi: POI?
+
+    
+    //ViewDidLoad
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updateViews()
+    }
+    
+    
+    // private func updateViews
+    private func updateViews() {
+        
+        guard let poi = poi else { return }
+        
+        locationLabel.text = poi.location
+        countryLabel.text = poi.country
+        
+        var clueText = ""
+        
+        for clue in poi.clues {
+            clueText += "∙ \(clue)\n"
+        }
+        
+        cluesTextView.text = clueText
```

Comment:
The app crashes here for me because there isn't a cluesTextView connected to the outlet. Be sure to add a textView in the storyboard and connect it. Then it should be good to go!